runelite-client/src/main/java/net/runelite/client/ui/ReducedFunctionalityPopup.java

```diff
@@ -0,0 +1,46 @@
+package net.runelite.client.ui;
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Dialog;
+import javax.swing.JDialog;
+import javax.swing.JLabel;
+import javax.swing.border.EmptyBorder;
+
+public class ReducedFunctionalityPopup
```

Comment:
Since this is just a dialog in itself, it may be worth extending JDialog instead of creating a new instance of JDialog in the constructor.